test/Confluent.Kafka.UnitTests/TopicPartitionOffsetError.cs

```diff
@@ -104,6 +106,8 @@ public void Properties()
         [Fact]
         public void ExplicitCast()
         {
+            Library.Load();
```

Comment:
Suggest adding a negative test that attempts to specify a non-existent librdkafka to make sure error reporting works as expected.
Also a positive test specifying an existing librdkafka.
(but neither of these obviously not here, somewhere else)

Reply:
I'll add a test for the negative case.